packages/flutter/test/widgets/generateTestSemanticsExpressionForCurrentSemanticsTree_test.dart

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Does this file need to be renamed to be in line with https://www.dartlang.org/guides/language/effective-dart/style#do-name-libraries-and-source-files-using-lowercase_with_underscores?

Reply:
This test is a little funky in that it tests one specific method that generates code, and this test simultaneously tests that the generated code _looks_ as expected and that it _behaves_ as expected (if Dart had `eval`, I probably wouldn't need this).

But, I should probably prefix the file name with the name of the file it's testing. So, partially done?